src/WinGet.RestSource.Infrastructure/Parameters/AppConfig/appconfig.int.centus.json

```diff
@@ -8,8 +8,14 @@
         "location": {
             "value": "centralus"
         },
-        "genevaLogging": {
-            "value": true
+        "featureFlags": {
+            "value": [
+                {
+                    "id": "GenevaLogging",
+                    "description": "Feature flag to use Geneva Monitoring.",
```

Comment:
>"description": "Feature flag to use Geneva Monitoring.", [](http://example.com/codeflow?start=20&length=56)

Do we need the description in every parameter file?

Can it just be defined in appconfig.json and here we just have the id and if it's enabled